pkg/handler/collector/s3/s3.go

```diff
@@ -89,8 +90,9 @@ func retrieve(s S3Collector, ctx context.Context, docChannel chan<- *processor.D
 			Format:   processor.FormatUnknown,
 			Encoding: bucket.ExtractEncoding(enc, item),
 			SourceInformation: processor.SourceInformation{
-				Collector: S3CollectorType,
-				Source:    "S3",
+				Collector:   S3CollectorType,
+				Source:      "S3",
```

Comment:
This source is wrong. Not something you did in this PR but has been wrong for some time. This should be `item`

Reply:
Fixed in b46408d5e5c9db78f79cb7da7c5b9962015c6df5.